feed/serializers.py

```diff
@@ -225,6 +225,7 @@ class Meta:
 
 class DisaggregationTypeSerializer(serializers.HyperlinkedModelSerializer):
     id = serializers.ReadOnlyField()
+    disaggregation_type = serializers.CharField(max_length=135)
```

Comment:
We already have '\_\_all__' in the Meta, so you don't need to add the `disaggregation_type` field here.

Reply:
This override is on purpose. I need to make this field mandatory, because in the model it's not.

Reply:
Ok